cmd/create/cluster/flag.go

```diff
@@ -25,6 +26,7 @@ type flag struct {
 
 func (f *flag) Init(cmd *cobra.Command) {
 	cmd.Flags().StringVarP(&f.Config, flagConfig, "g", "", `The path to the file containing API endpoints and tokens for each provider.`)
+	cmd.Flags().StringVarP(&f.Kubeconfig, flagKubeconfig, "k", "", `The path to the directory containing the kubeconfigs for provider control planes.`)
```

Comment:
There is a new flag so we can talk to the management cluster and fetch the `Organizations`. We'd need to update the existing tekton `Task`.